cronjobs/fetch_marc_updates.py

```diff
@@ -114,7 +114,8 @@ def RetrbinaryCallback(chunk):
     else:
         return None
 
-
+# Cumulatively saves downloaded data to external location to have complete trace of 
+# downloaded data. Thus, a complete data should be resconstructible  
```

Comment:
As "data" is a plural noun, there is no such thing as "a data" (It would be "a datum".)
a data -> the data
A period at the end is missing.